### PR TITLE
small typo in documentation for attribute-picker

### PR DESCRIPTION
**Pull request by abcotter**
just a small typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ An autocompleting dropdown to choose one or more new or pre-existing attributes
 **Usage:**
 ```html
 <script type="module">
-    import '@brightspace-ui-labs/attribute-picker/attribute-picker.js';
+    import '@brightspace-ui-labs/multi-select/attribute-picker.js';
```
Comment (dlockhart) [inline README.md:65]: Haha technically the lines below are also `<d2l-labs-attribute-picker>`... probably a copy-paste thing.

**Review by tenbradley — APPROVED** [review]
Seems legit.

**Comment by abcotter** [comment]
> Seems legit.

are ya'll a 2 approves team? or am I good to merge? :shipit:

**Comment by tenbradley** [comment]
> are ya'll a 2 approves team? or am I good to merge? :shipit:

We only require one approve. Ship it real good.

**Pull request merged**